vendor/assets/javascripts/checkbox_modal.js

```diff
@@ -0,0 +1,26 @@
+// allow a cancel button to work properly on checkbox modal
+Cmsify.CheckboxModal = function(el) {
+  this.$el = $(el);
+  this.$el.addClass('uk-modal');
+  this.checked = {};
+  this.modal = UIkit.modal(el);
+  this.modal.on('show.uk.modal', function() {
+    this.storeCheckboxes();
+  }.bind(this));
+  this.$el.find('.js-cancel-' + this.$el.attr('id')).each(function(index, cancel) {
+    $(cancel).on('click', this.loadCheckboxes.bind(this)); 
```

Comment:
Yeah, I actually wasn't sure on this. I could bind it to the `hide.uk.modal` and then make the Done button run storeCheckboxes before closing (so that it would be the latest results of the checkboxes). I guess I assumed that if you closed the modal by clicking outside of the modal box you would want your changes saved but that might not have been the right assumption to make.